common/core/desktop/build.bat

```diff
@@ -0,0 +1,100 @@
+@echo off
+
+setlocal enabledelayedexpansion
+
+if "%1"=="" goto help
+if "%1"=="all" goto all
+if "%1"=="x86" goto build
+if "%1"=="x64" goto build
+
+echo "Invalid parameter."
+goto help
+
+rem ----------------------------------
+
+:help
+echo Usage: build-core.bat x86^|x64^|all [-c]
+echo -c will leave your environment configured for Visual Studio for selected platform.
+echo -c can be used only with x86 and x64 options
+goto :eof
+
+rem ----------------------------------
+
+:all
+
+setlocal
+cd %KEYMAN_ROOT%\common\core\desktop
+cmd /c build.bat x86
+if errorlevel 1 exit /b !errorlevel!
+
+cd %KEYMAN_ROOT%\common\core\desktop
+cmd /c build.bat x64
+if errorlevel 1 exit /b !errorlevel!
+
+goto :eof
+
+rem ----------------------------------
+
+:build
+
+if "%2"=="-c" goto :setup
+
+set ARCH=%1
+echo Building Keyman Core for Windows !ARCH!
+
+echo === Locating Visual Studio ===
+
+rem From https://github.com/microsoft/vswhere
+for /f "usebackq tokens=*" %%i in (`..\..\..\resources\build\vswhere -latest -requires Microsoft.Component.MSBuild -find **\vcvarsall.bat`) do (
```

Comment:
Should this be relative to `%KEYMAN_ROOT%?`

Reply:
This works without needing to use `%KEYMAN_ROOT%` -- for now...